src_c/rwobject.c

```diff
@@ -662,7 +667,7 @@ _pg_rw_read(SDL_RWops *context, void *ptr, size_t size, size_t maxnum)
 {
     pgRWHelper *helper = (pgRWHelper *)context->hidden.unknown.data1;
     PyObject *result;
-    ssize_t retval;
+    Py_ssize_t retval;
```

Comment:
This looks like a bug? The function returns size_t, but the retval is ssize_t. Should they both be size_t instead?

Reply:
ps. As a side note - I'm not sure if this is the kind of thing that should be reported to python as breakage? I'm guessing this type will come back before the official release since that type is fairly common.

Reply:
AFAIK, `Py_ssize_t` is just an alias for `ssize_t` when `ssize_t` is available, and when it is not, defines an equivalent implementation of the same. So basically, `Py_ssize_t` should always be defined